scripts/build_locally.py

```diff
@@ -73,6 +73,7 @@ def run(
         nargs="?",
         const="Debug",
         default="Release",
+        choices=["Release", "Debug"],
```

Comment:
If the intention was to support `--debug` option with no arguments, the line should be

```python
 driver.add_argument("--debug",
     type=str,
     default="Release",
     const="Debug",
     action="store_const"
  )
```

For example:

```python
In [10]: parser.parse_args([])
Out[10]: Namespace(debug='Release')

In [11]: parser.parse_args(["--debug"])
Out[11]: Namespace(debug='Debug')
```

If the intent is to specify the built type explicitly:

```python
driver.add_argument("--build_type",
    type=str,
    default="Release",
    choices=["Release", "Debug"]
)
```

The usage:

```python
In [15]: parser.parse_args([])
Out[15]: Namespace(debug='Release', build_type='Release')

In [16]: parser.parse_args(["--build_type=Debug"])
Out[16]: Namespace(debug='Release', build_type='Debug')
```

Reply:
Thanks for these examples. I wanted to implement support  `--debug` option without arguments.

Reply:
Ok, great. Then please make the change. Notice that `narg` parameter is no needed to accomplish that.